### PR TITLE
Blog for JBDS 8.1.0.CR1

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Note: JBoss Tools 4.2.3 and JBoss Developer Studio 8.1.0 will become available in Marketplace only for Final releases at early April
+
+== What is New?
+
+We are getting close to the final release. So, we focused on bug fixing in this release.
+
+== Upgraded Forge 2 Runtime
+
+Besides big fixing the Forge 2 runtime was upgraded to 2.15.1.Final. See the release notes link:https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12311820&version=12326733[here].
```
Comment (maxandersen) [inline blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc]: big-> bug

File: blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
+= JBoss Developer Studio 8.1.0.CR1 and JBoss Tools 4.2.3.CR1
```
Comment (maxandersen) [inline blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc]: filename should have devstudio in it. we dont use JBDS for anything but the jira key.

Comment (nickboldt) [inline blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc]: Should we also fix the Beta1 blog then? https://github.com/nickboldt/jbosstools-website/blob/release423CR1/_config/products.yml#L70

**Comment by maxandersen** [comment]
looks good. +1

File: blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc (hunk 2 above)
Comment (maxandersen) [inline blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc]: that url is already public so no.

**Comment by alexeykazakov** [comment]
Rebased and pushed to master and production.

**Pull request closed without merging**